src/Components/BehaviorsModal/Internal/Widgets/GaugeWidget/GaugeWidget.tsx

```diff
@@ -17,6 +18,7 @@ const GaugeWidget: React.FC<IProp> = ({ widget }) => {
     const label = widget.widgetConfiguration.label;
     const units = widget.widgetConfiguration.units || '';
     let value = 0;
+    let format = '';
```

Comment:
nit: renaming this var to reflect that it is a formatted "value", not format itself.
```suggestion
    let formatedValue = '';
```